src/clean.js

```diff
@@ -0,0 +1,93 @@
+const { contentfulManagementClient, contentfulPreviewClient, defaultLocale } = require('./config');
+
+const deleteEntry = async(id) => {
+  const contentfulManagementConnection = await contentfulManagementClient.connect();
```

Comment:
Move this to a global and connect in the `clean` function so that it does not connect on each call to this function.

This may speed up the script.